extensions/wikia/AdEngine/js/wad/btRecLoader.js

```diff
@@ -14,37 +14,53 @@ define('ext.wikia.adEngine.wad.btRecLoader', [
 		placementsMap = {
 			TOP_LEADERBOARD: {
 				uid: '5b33d3584c-188',
-				style: 'margin: 10px 0;'
+				style: 'margin:10px 0; z-index:100;',
```

Comment:
100 should be good enough and it's in the WDS scale: https://github.com/Wikia/app/blob/dev/extensions/wikia/DesignSystem/node_modules/design-system/dist/scss/wds-variables/_z-index-scale.scss#L11

However, I skimmed through our ads code and we use some outdated vars: https://github.com/Wikia/app/blob/1e4ea22073c29ec97beeccda86d068915366d0c5/skins/oasis/css/core/ads-porvata.scss#L4

Anyway, I don't think there is a way to re-use SASS vars in JS, so I think we need to leave it as-is here. Unless there is a way to re-use it...

Reply:
`100` is the exact same value like for `TOP_RIGHT_BOXAD`. And I also don't know how to connect these places with additionally keeping in mind that we need to be compatible with Blockthrough rules and not to make AdBlock developers job easier :(